### DISCUSSION TITLE
Dudas semana 3- Css

### DISCUSSION THREAD

**Discussion by saraivetesigv** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#270] [Q&A]
Buenas tardes.
Yo voy empezando con la practica de la semana 3. 
Mi duda a lo mejor es algo simple, que no estoy viendo. 
Pero a ver si alguien me ayuda a aclararla.

La pagina que puso el commander, es solo pagina principal o se va deslizando las pestañas siguientes.

**Comment by Ingrzs** [comment]
Cuales pestañas siguientes?
O quisiste decirte que la página se desliza hacia abajo?

Eso se hace en automático les vas añadiéndo elementos y la pagina se "agranda hacia abajo"

La práctica de la página de la vacuna es solo una, la principal que subio en la imagen, que tu quieras que tu página tenga otras paginas (clik en algun boton(navbar)) es desicion tuya o bonus, el mc aclaro que solo la quiere similar ala pagina.

**Reply by saraivetesigv** [reply]
si me referia a las pestañas database y las siguientes. Pero ya me queda mas claro.

**Comment by IsaiVelasco** [comment]
A mi entender sí, la página que puso el Comander es solo la principal. Porque más adelante donde menciona los bonus, dice que puedes agregar otros diseños y páginas adicionales.

**Reply by saraivetesigv** [reply]
Gracias entonces asi lo hare

**Comment by JuanSalvador73** [comment]
Básate en lo que nos dió, está en uno agregar algo mas a ese sitio. Yo personalmente lo recomiendo, ya que se trata de hacer algo semejante pero con estilo propio.
